lib/install.ps1

```diff
@@ -86,9 +86,13 @@ function Find-Manifest($app, $bucket) {
         $manifest = url_manifest $url
     } else {
         # check buckets
-        $manifest, $bucket = find_manifest $app $bucket
+        if (!$bucket) {
+            $manifest, $bucket = find_manifest $app $bucket
```

Comment:
I do not understand. Why are you passing $bucket when it is $null or do not exists

```suggestion
            $manifest, $bucket = find_manifest $app
```